src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLastLineCommentCheck.java

```diff
@@ -106,7 +106,7 @@ void method7(int i, int j, boolean cond) {
           switch (i){
           case 5: // violation 'Fall\ through from the last branch of the switch statement'
               i++;
-              /* block */ i++; /* fallthru */ // comment
+              /* block */ i++; /* Fallthru */ // comment
```

Comment:
Restore as it was before

Reply:
@romani what about these

Reply:
Make it fallthru, as it was before.

Reply:
> Make it fallthru, as it was before.

Violation also need to be removed?

Reply:
violation i always representation of actual behavior.
if there is violation from checkstyle, then we should put `// violation`.
if no violation, we remove comment.

Again: Code(even target comments) stays the same and only "// violation " comments changing their place or added or removed.